style.css

```diff
@@ -0,0 +1,150 @@
+
+body{
+    background-color: #eceef1;
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+    font-family:  Helvetica, sans-serif;
+    color: #4c4c4c;
+}
+
+header{
+    display: flex;
+    justify-content: flex-end;
+    font-size: 13px; 
+    align-items: center;
+}
+
+
+#action-btn{
+    background-color: #0077c5;
+    color: white;
+    border-radius: 3px;
+    border:1px gray;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.main-logo{
+    display: flex;
+    justify-content: center;
+    margin:15px;
+}
+
+.section-stuff{
+    display: flex;
+    justify-content: center;
+    flex-direction: row;
+    width: 100%;
+
+}
+
+.small-logos{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+.small-logos>.logos{
+    display: flex;
+    align-items: center;
+    margin: 20px 5px 20px 5px ;
+}
+.logos>span{
+    font-size: 20px;
+    color: darkgray;
+    padding: 5px;
+}
+
+a {
+    color: #0077c5;
+    text-decoration: none;
+}
+
+.form{
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    width: 400px;
+    border: 1px darkgray solid ;
+    border-radius: 2px;
+    background-color: white;
+
+
+}
+
+.form>div, p{
+    text-align: center;
+}
+
+.form>input,label,button{
+    margin: 5px 20px;
+    border-radius: 3px;
+    height: 34px;
+    /* width: 320px; */
```

Comment:
Kindly delete commented code. Use git history if you want to revert back your code